### PR TITLE
Make SaladBind easier to set up

### PR DESCRIPTION
**Pull request by TheBlueBurger**

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -60,10 +60,10 @@ if (process.argv[2]) {
 
 (async() => {
 	update.updateCheck.then(() => {
+		let firstTime = fs.existsSync("./data");
 		if (!CLImode) {
-			console.log(chalk.bold.green(`SaladBind v${packageJson.version}`))
+			if(!firstTime) console.log(chalk.bold.green(`SaladBind v${packageJson.version}`));
```
Comment (Vukkyy) [inline src/index.js:65]: Do we need this? Can't we just put the console.log above the menu(); on line 69?

File: src/setup.js [hunk 2]
```diff
@@ -34,6 +36,15 @@ async function continueSetup(clear) {
 		console.clear();
 		console.log(chalk.bold.cyan(`Configure SaladBind`))
 	}
+	if(firstTime) {
+		console.log(`
+${chalk.greenBright.bold("Welcome to SaladBind!")}
+This is a program that makes it easier to select miner, algorithm and pool for Salad! All of the money you mine using SaladBind goes to Salad (except for the miner and pools devfees)
+All salad boosts and XP will work in SaladBind.
```
Comment (Vukkyy) [inline src/setup.js:43]:
```suggestion
This is a program that makes it easier to select miner, algorithm and pool for Salad! All of the money you mine using SaladBind goes to Salad, and all Salad boosts and XP will work in SaladBind.
```

File: src/setup.js [hunk 3]
```diff
@@ -34,6 +36,15 @@ async function continueSetup(clear) {
...
+
+Discord Rich Presence Means that your discord friends and people in Discord servers are going to see that you use SaladBind. They will see the SaladBind version, miner, algorithm and pool in your Discord profile.
```
Comment (Vukkyy) [inline src/setup.js:45]:
```suggestion
Discord Rich Presence means that your discord friends and people in Discord servers are going to see that you use SaladBind. They will see your SaladBind version, miner, algorithm, and pool on your Discord profile.
```

**Review by Vukkyy — CHANGES_REQUESTED** [review]
uh oh the vukky is slightly picky :D

**Review by Vukkyy — APPROVED** [review]
lgtm!

**Comment by TheBlueBurger** [comment]
Merging if nobody finds any issues

**Comment by Vukkyy** [comment]
lol

**Comment by memedmusic** [comment]
#69 lol

**Pull request merged**